sdk/core/Azure.Core.TestFramework/src/Instrumentation/ManagementInterceptor.cs

```diff
@@ -24,50 +24,24 @@ public ManagementInterceptor(ClientTestBase testBase)
 
         public void Intercept(IInvocation invocation)
         {
-            bool modifiedAskToWait = false;
-
-            if (_testMode == RecordedTestMode.Playback && IsLro(invocation.Method.ReturnType))
```

Comment:
I think we still need this otherwise we will apply zero wait during recording won't we?

Reply:
Good catch!